src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts

```diff
@@ -1,13 +1,65 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 - 2023 Gemeente Amsterdam
 import { mockFiltersShort } from '../__test__/mock-filters'
-import { mockIncidentsShort } from '../__test__/mock-incidents'
+import { mockIncidentsWithoutIcon } from '../__test__/mock-incidents-without-icon'
 import { getFilteredIncidents } from './get-filtered-incidents'
 
+const mockFiltersAllActive = mockFiltersShort.map((filter) => {
+  if (filter.subCategories) {
```

Comment:
I would check on Array.isArray(filter.subCategories). Seems a little effort.

Reply:
or is it typed?

Reply:
it is typed indeed